ext/semian/semian.c

```diff
@@ -35,11 +35,13 @@ static ID id_timeout;
 static VALUE eSyscall, eTimeout, eInternal;
 static int system_max_semaphore_count;
 
-static const int kIndexTickets = 0;
-static const int kIndexTicketMax = 1;
-static const int kIndexLock = 2;
-
-static const int kNumSemaphores = 3;
+typedef enum
+{
+  sem_tickets,     // semaphore for the tickets currently issued
+  sem_max_tickets, // semaphore to track the maximum number of tickets available for issue
```

Comment:
@sirupsen hadn't pinged you yet because still working on this, but yeah, my first question was going to be "what should we name these things"